{{cookiecutter.project_slug}}/requirements.txt

```diff
@@ -1,4 +1,5 @@
 # This file defines the requirements for a productive project deployment.
 # It should not contain dependencies for tests, developer tools, etc. (see requirements-dev.txt)
-{% if cookiecutter.config_file == 'hocon' %}pyhocon~=0.3.54{% elif cookiecutter.config_file == 'yaml' %}PyYAML~=5.3.1{% endif %}
-{% if cookiecutter.create_cli == 'yes' %}typer~=0.2.1{% endif %}
+{% if cookiecutter.config_file == 'hocon' %}pyhocon~=0.3.58{% elif cookiecutter.config_file == 'yaml' %}PyYAML~=5.4.1{% endif %}
+{% if cookiecutter.create_cli == 'yes' %}typer~=0.4.0{% endif %}
+wheel~=0.37.0
```

Comment:
@klamann, I haven't really seen this done much so there might a reason why it is a bad idea. But I have encountered situations where `wheel` was not installed, making installation of other packages less efficient or causing them to fail. I am sure you have an opinion on this :)

Reply:
I sometimes put `wheel` into the `requirements-dev.txt`, but even then, if you install all your dependencies with a single call, the wheel package will only be available after you have (inefficiently) installed all your dependencies. Not sure what's the best approach here, to be honest.

Reply:
Thanks for your thoughts! I put `wheel` into `requirements-dev.txt`. At least for me that would help, since I often set up a venv when I start working on a project and add dependencies to `requirements.txt` as I start using them. And having `wheel` included in the template might help remember installing it. ;)